### ISSUE TITLE
Charts colors customization

### ISSUE THREAD

**Issue by vmato** [pgadmin-org/pgadmin4#7588]
v8.8 brought new color scheme for charts. Rollbacks and deletes are now green as if it was something desired. Is there a way to customize these colors?

**Comment by adityatoshniwal** [comment]
Yes, colors are changed to use same color palette in pgAdmin. Colors here are only to differentiate and have no meaning.

**Comment by vmato** [comment]
As of design colors always have meaning

**Comment by vmato** [comment]
As of design, colors always have meaning
